src/coreComponents/physicsSolvers/NonlinearSolverParameters.cpp

```diff
@@ -187,44 +188,50 @@ void NonlinearSolverParameters::postProcessInput()
-  GEOS_LOG_RANK_0( GEOS_FMT( "  Minimum iterations = {}", m_minIterNewton ) );
-  GEOS_LOG_RANK_0( GEOS_FMT( "  Maximum allowed residual norm = {}", m_maxAllowedResidualNorm ) );
-  GEOS_LOG_RANK_0( GEOS_FMT( "  Allow non-converged = {}", m_allowNonConverged ) );
-  GEOS_LOG_RANK_0( GEOS_FMT( "  Time step decrease iterations limit = {}", m_timeStepDecreaseIterLimit ) );
-  GEOS_LOG_RANK_0( GEOS_FMT( "  Time step increase iterations limit = {}", m_timeStepIncreaseIterLimit ) );
-  GEOS_LOG_RANK_0( GEOS_FMT( "  Time step decrease factor = {}", m_timeStepDecreaseFactor ) );
-  GEOS_LOG_RANK_0( GEOS_FMT( "  Time step increase factor = {}", m_timeStepDecreaseFactor ) );
-  GEOS_LOG_RANK_0( GEOS_FMT( "  Time step cut factor = {}", m_timeStepCutFactor ) );
-  GEOS_LOG_RANK_0( GEOS_FMT( "  Maximum time step cuts = {}", m_maxTimeStepCuts ) );
-  GEOS_LOG_RANK_0( GEOS_FMT( "  Maximum sub time steps = {}", m_maxSubSteps ) );
-  GEOS_LOG_RANK_0( GEOS_FMT( "  Maximum number of configuration attempts = {}", m_maxNumConfigurationAttempts ) );
-  GEOS_LOG_RANK_0( GEOS_FMT( "  Coupling type = {}", EnumStrings< CouplingType >::toString( m_couplingType ) ) );
+  tableData.addRow( "Norm type (flow solver)", m_normType );
+  tableData.addRow( "Minimum residual normalizer", m_minNormalizer );
+  tableData.addRow( "Convergence tolerance", m_newtonTol );
+  tableData.addRow( "Maximum iterations", m_maxIterNewton );
+  tableData.addRow( "Minimum iterations", m_minIterNewton );
+  tableData.addRow( "Maximum allowed residual norm", m_maxAllowedResidualNorm );
+  tableData.addRow( "Allow non-converged", m_allowNonConverged );
+  tableData.addRow( "Time step decrease iterations limit", m_timeStepDecreaseIterLimit );
+  tableData.addRow( "Time step increase iterations limit", m_timeStepIncreaseIterLimit );
+  tableData.addRow( "Time step decrease factor", m_timeStepDecreaseFactor );
+  tableData.addRow( "Time step increase factor", m_timeStepDecreaseFactor );
+  tableData.addRow( "Time step cut factor", m_timeStepCutFactor );
+  tableData.addRow( "Maximum time step cuts", m_maxTimeStepCuts );
+  tableData.addRow( "Maximum sub time steps", m_maxSubSteps );
+  tableData.addRow( "Maximum number of configuration attempts", m_maxNumConfigurationAttempts );
+  tableData.addRow( "Coupling type", m_couplingType );
   if( m_couplingType == CouplingType::Sequential )
   {
-    GEOS_LOG_RANK_0( GEOS_FMT( "  Sequential convergence criterion = {}", EnumStrings< SequentialConvergenceCriterion >::toString( m_sequentialConvergenceCriterion ) ) );
-    GEOS_LOG_RANK_0( GEOS_FMT( "  Subcycling = {}", m_subcyclingOption ) );
+    tableData.addRow( "Sequential convergence criterion", m_sequentialConvergenceCriterion );
+    tableData.addRow( "Subcycling", m_subcyclingOption );
   }
+  TableLayout const tableLayout = TableLayout( {
+      TableLayout::ColumnParam{"Parameter", TableLayout::Alignment::left},
+      TableLayout::ColumnParam{"Value", TableLayout::Alignment::center},
```

Comment:
My personal preference would be left alignment for both columns.

Reply:
changed